packages/react-components/src/Concepts/Positioning/PositioningDefault.stories.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import { PositioningProps } from '@fluentui/react-positioning';
+import { PositionedComponent } from './utils.stories';
+
+export const Default = (props: PositioningProps) => {
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/132231002-45b47bc7-2d07-4aa4-891a-23b023a2888a.png)

As in other PRs: not all controls are useful, can we disable them where options don't do what we expect?